src/middle/Expr.ml

```diff
@@ -221,7 +222,7 @@ module Helpers = struct
       | Var _ | TupleProjection _ -> Fixed.Pattern.Indexed (e, [i])
       | Indexed (e, indices) -> Indexed (e, indices @ [i])
       | _ ->
-          Common.FatalError.fatal_error_msg
+          Common.ICE.internal_compiler_error
```

Comment:
```suggestion
          ICE.internal_compiler_error
```
There's `open Common` at the start of this file and the previous `ICE.internal_compiler_error` omits `Common.` prefix so I'd like to be consistent.